libs/stringtable/src/rapify.rs

```diff
@@ -0,0 +1,161 @@
+use crate::{Key, Project};
+use hemtt_workspace::WorkspacePath;
+use tracing::warn;
+
+static ALL_LANGUAGES: [&str; 14] = ["English", "Czech", "French", "Spanish", "Italian", "Polish", "Portuguese", "Russian", "German", "Korean", "Japanese", "Chinese", "Chinesesimp", "Turkish"];
```

Comment:
Since 2.18 also
Dutch, Finnish, Ukrainian, Swedish, Norwegian, Danish.

See `Arma 3/dta/languageList.bin`

Reply:
https://community.bistudio.com/wiki/Stringtable.xml

not sure if we should add Icelandic, Slovak?  They aren't in languageList.bin
If we don't include a language they get empty strings (there is no fallback)
Don't think it would hurt?

Reply:
I would add them, they're already in the big struct